net/mwan3/Makefile

```diff
@@ -26,6 +26,7 @@ define Package/mwan3
      +iptables-mod-conntrack-extra \
      +iptables-mod-ipopt \
      +jshn
+   PKG_CONFIG_DEPENDS:=IPV6
```

Comment:
We need this for the C code, right?
If at all, then it must not be in the package definition but after the include in the top level. Also I think the vaule should be called CONFIG_IPV6.

Reply:
yes, it's needed to tell the build system to recompile if CONFIG_IPV6 is changed. You are right about the naming and location of this. will fix.